package.json

```diff
@@ -55,7 +55,8 @@
     "docs": "node scripts/generate-readme.js",
     "simple-integration-test": "jest --maxWorkers=5 simple-suite",
     "complex-integration-test": "jest --maxWorkers=5 integration",
-    "postinstall": "node ./scripts/postinstall.js"
+    "postinstall": "node ./scripts/postinstall.js",
+    "pre-release": "node ./scripts/pre-release.js"
```

Comment:
Another question: Do we need to add this script to the `package.json` `files` array?